compiler/env/OMRAOTStorageInterface.cpp

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+#include "env/AOTStorageInterface.hpp"
+#include "infra/Assert.hpp"
+
+TR::AOTStorageInterface *
+OMR::AOTStorageInterface::self()
+   {
+   return static_cast<TR::AOTStorageInterface *>(this);
+   }
+
+uint8_t* OMR::AOTStorageInterface::loadEntry(const char* key )
+   {
+   TR_UNIMPLEMENTED();
+   return 0;
+   }
+
+void OMR::AOTStorageInterface::storeEntry(const char* key, void* data, uint8_t size)
+   {
+   TR_UNIMPLEMENTED();
+   }
+
+uint8_t* OMR::AOTStorageInterface::allocateMemoryInCache(uint8_t size)
```

Comment:
since the class is the generic "AOTStorageInterface" I don't think this function should assume the storage target is a cache. Perhaps`allocateStorageEntry()` or even `allocateEntry()` since it's already a "storage" class?

Reply:
Thank you, I've corrected and updated the PR.